packages/app/src/components/prague/GuideDrawer.tsx

```diff
@@ -45,249 +33,114 @@ const Guide = () => {
-                      <AccordionIcon></AccordionIcon>
-                    </Text>
-                  </VStack>
-                  <AccordionPanel pb={4}>
-                    <VStack spacing={6} alignItems="flex-start">
-                      <Text fontSize="xs">
-                        {t(
-                          "Porto team will send your MACI key the email address you used to register for the event."
-                        )}
-                      </Text>
-                    </VStack>
-                  </AccordionPanel>
-                </AccordionItem>
-              </ListItem>
-              <ListItem>
-                <AccordionItem border="none">
-                  <VStack>
-                    <Text
-                      px={0}
-                      textAlign={"left"}
-                      as={AccordionButton}
-                      fontSize="md"
-                    >
-                      {t("Step 2: Connect your wallet and enter your MACI key")}
-                      <AccordionIcon></AccordionIcon>
-                    </Text>
-                  </VStack>
-                  <AccordionPanel pb={4}>
-                    <VStack spacing={6} alignItems="flex-start">
-                      <Text fontSize="xs">
-                        <UnorderedList>
-                          <ListItem mb={4}>
-                            {t(
-                              "Click on ‘START’ and it’ll direct you to the voting website."
-                            )}
-                          </ListItem>
-                          <ListItem mb={4}>
-                            {t(
-                              "Tap 'Connect' and follow the instructions to successfully connect your Wallet, (WalletConnect is also available)."
-                            )}
-                          </ListItem>
-                          <ListItem>
-                            {t(
-                              "Once your wallet is connected, enter your MACI key in the field below and press 'Save'."
-                            )}
-                          </ListItem>
-                        </UnorderedList>
-                      </Text>
-                    </VStack>
-                  </AccordionPanel>
-                </AccordionItem>
-              </ListItem>
-            </UnorderedList>
+            <AccordionItem border="none">
+              <VStack>
+                <Text px={0} textAlign={"left"} as={AccordionButton} fontSize="md">
+                  Step 1: Download a wallet that supports Gnosis Chain (xDAI)
+                  <AccordionIcon></AccordionIcon>
+                </Text>
+              </VStack>
+              <AccordionPanel pb={4}>
+                <VStack spacing={6} alignItems="flex-start">
+                  <iframe title="guide" src="https://scribehow.com/page-embed/Mobile_Wallet_Set_up__OZiM3PehSGmXB73cXh85KQ" width="315" height="640" allowFullScreen frameBorder="0"></iframe>
```

Comment:
So my take would be to link out to the guides vs. embedding them. Given it's a side panel, it's too small to view:
![Image 2023-03-20 at 5 24 28 PM](https://user-images.githubusercontent.com/8097623/226420994-0f71c0c1-0c00-4b41-b4d8-bb4a0c172703.jpg)

Reply:
With links we could also allow the user to pick the mobile or desktop guide. e.g.

`Step 1: Download a wallet that supports Gnosis Chain (xDAI)`

`View our [mobile guide](link) or our [desktop guide](link) depending on the device you're using`